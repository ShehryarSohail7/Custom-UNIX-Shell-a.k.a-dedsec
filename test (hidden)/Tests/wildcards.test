ls *
ls *.c
echo ../src/*.c
wc -l ../src/*.c ../include/*.h
ls -a Tests/*.test | sort -k5 -n