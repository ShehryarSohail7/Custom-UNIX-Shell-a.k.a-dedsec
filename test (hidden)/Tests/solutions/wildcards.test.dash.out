config.json
find.log
names.txt
test.py

&&:

*:

;:

Tests:
aliases.test
aliases.test.custom
builtins.test
builtins_one.hidden
builtins_one.hidden.custom
builtins_two.hidden
builtins_two.hidden.custom
chaining.hidden
chaining.test
exec_only.hidden
ioredir.test
ioredir_one.hidden
ioredir_two.hidden
ioredir_two.hidden.custom
pipeline.test
pipeline_ioredir.hidden
pipeline_one.hidden
pipeline_two.hidden
quotes.test
quotes.test.custom
simple.test
test_output
wild_chaining.hidden
wild_chaining.test
wildcards.test
wildcards_one.hidden

cp:

ls:

test_dir:
../src/main.c
  819 ../src/main.c
   88 ../include/log.h
   27 ../include/utils.h
  934 total
Tests/aliases.test
Tests/builtins.test
Tests/chaining.test
Tests/ioredir.test
Tests/pipeline.test
Tests/quotes.test
Tests/simple.test
Tests/wild_chaining.test
Tests/wildcards.test
